### ISSUE TITLE
Handle nested lists in filtering in a nicer way

### ISSUE THREAD

**Issue by scottrobertson** [DamascenoRafael/reminders-menubar#36]
Right now it does not show the top level group in the filtering list, which makes it a little confusing.

Depending on what is available in SwiftUI, a fix for this could either be adding a "Group" to the dropdown of some sort, or failing that, just prefix the list with the group: "Group -> List" in the filter list.

**Comment by DamascenoRafael** [comment]
Good point, I didn't know about the possibility of creating a group of lists 😆 

I still haven't found a way to access this property.
 I commented on [this thread on Apple forum](https://developer.apple.com/forums/thread/683611) and I'm waiting to see if there's an update.

If I find anything new I'll post it here.

**Comment by DamascenoRafael** [comment]
Another thread indicating that unfortunately it is not yet available in the Apple API:
[Fetching Reminder Subtasks (+ Other Properties)?](https://developer.apple.com/forums/thread/655905)

I'll check if there's anything new in the new macOS 13.

**Comment by DamascenoRafael** [comment]
In #90 the URL information saved in a reminder was not publicly available in an Apple API, but despite that I found a workaround on the internet to access that value and it worked.

I believe the same can be done for nested-lists/subtasks 🤞 

This was the change to access the URL info:
https://github.com/DamascenoRafael/reminders-menubar/commit/5f5d96c51cfbaeeb97ae96debe0f3fc4515d9024#diff-3b9abc59732ccfa44415fc0a061aa164825956389e41c5be9af3527562df22ac

It is still necessary to investigate the reminder objects to try to fetch this information.

I'll try to find some free time to investigate this and add this improvement.
Thanks again for the suggestions to make this app better.

**Comment by pottersfield** [comment]
With the announcement of interactive widgets for iOS, iPadOS and macOS did you found any changes in their API for subtasks? The native widgets only show the number of subtasks but not the subtasks itself unfortunately. 
Thx for the well made app! ✌️

**Comment by DamascenoRafael** [comment]
Updates on handling **subtasks**.
I'm currently working on this, and I believe I can make it available in the next version of Reminders MenuBar.

Here's a sneak peak of it working:
<img width="775" alt="Screenshot 2023-09-24 at 22 39 34" src="https://github.com/DamascenoRafael/reminders-menubar/assets/9842644/b23c70df-65f0-43ad-a1ab-965adceded57">

**Comment by DamascenoRafael** [comment]
> With the announcement of interactive widgets for iOS, iPadOS and macOS did you found any changes in their API for subtasks? The native widgets only show the number of subtasks but not the subtasks itself unfortunately. Thx for the well made app! ✌️

@pottersfield Apple still doesn't provide an API for subtasks, but I'm doing a workaround to access this information (as shown in the picture in the post above).

**Comment by timlance** [comment]
looks good!

**Comment by ianb-mp** [comment]
Just wondering if there's any update on this?
